src/ontology/cl-edit.owl

```diff
@@ -30811,12 +30813,12 @@ AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:16230044") rdfs:comment
 AnnotationAssertion(rdfs:label obo:CL_4030025 "renal cortical fibroblast")
 EquivalentClasses(obo:CL_4030025 ObjectIntersectionOf(obo:CL_0000057 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0005270)))
 
-# Class: obo:CL_4030026 (BEST4+ intestinal epithelial cell, human)
+# Class: obo:CL_4030026 (BEST4+ enterocyte)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An absorptive epithelial cell of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref <https://orcid.org/0000-0002-7431-4139>) Annotation(oboInOwl:hasDbXref "PMID:24223998") Annotation(oboInOwl:hasDbXref "PMID:35176508") obo:IAO_0000115 obo:CL_4030026 "An enterocyte of the human intestine expressing bestrophin-4 (BEST4) calcium-activated ion channels.")
```

Comment:
consider to add the broad synonym
BEST4+ epithelial cell https://www.nature.com/articles/s41586-021-03852-1
BEST4+ cell
and
The related synonym
BEST4-expressing enterocyte (same paper)

Reply:
we decided to add as exact

Reply:
@JABelfiore Have these terms been added as exact synonyms? I can only see BEST4+ intestinal epithelial cell, human as exact synonym